core/assets/bundles/bundle_ru.properties

```diff
@@ -69,7 +69,7 @@ map.delete = Вы действительно хотите удалить кар
 level.highscore = Рекорд: [accent]{0}
 level.select = Выбор карты
 level.mode = Режим игры:
-showagain = Не показывать снова до следующей сессии
+showagain = Не показывать до следующей сессии
```

Comment:
String should be deleted.
![image](https://user-images.githubusercontent.com/31485341/80311775-481a4680-87ea-11ea-93fe-213e496a6abf.png)